app/src/main/java/org/jellyfin/androidtv/browsing/HomeFragmentFooterRow.java

```diff
@@ -0,0 +1,76 @@
+package org.jellyfin.androidtv.browsing;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.widget.Toast;
+
+import org.jellyfin.androidtv.R;
+import org.jellyfin.androidtv.TvApp;
+import org.jellyfin.androidtv.presentation.CardPresenter;
+import org.jellyfin.androidtv.presentation.GridButtonPresenter;
+import org.jellyfin.androidtv.settings.SettingsActivity;
+import org.jellyfin.androidtv.startup.SelectUserActivity;
+import org.jellyfin.androidtv.ui.GridButton;
+
+import androidx.leanback.widget.ArrayObjectAdapter;
+import androidx.leanback.widget.HeaderItem;
+import androidx.leanback.widget.ListRow;
+import androidx.leanback.widget.OnItemViewClickedListener;
+import androidx.leanback.widget.Presenter;
+import androidx.leanback.widget.Row;
+import androidx.leanback.widget.RowPresenter;
+
+public class HomeFragmentFooterRow extends HomeFragmentRow implements OnItemViewClickedListener {
+    private static final int LOGOUT = 0;
+    private static final int SETTINGS = 1;
+
+    private Activity activity;
+
+    public HomeFragmentFooterRow(Activity activity) {
+        this.activity = activity;
+    }
+
+    @Override
+    public void addToRowsAdapter(CardPresenter cardPresenter, ArrayObjectAdapter rowsAdapter) {
+        HeaderItem header = new HeaderItem(rowsAdapter.size(), activity.getString(R.string.lbl_settings));
+        GridButtonPresenter presenter = new GridButtonPresenter();
+
+        ArrayObjectAdapter adapter = new ArrayObjectAdapter(presenter);
+        adapter.add(new GridButton(SETTINGS, activity.getString(R.string.lbl_settings), R.drawable.tile_settings));
+        adapter.add(new GridButton(LOGOUT, activity.getString(R.string.lbl_logout), R.drawable.tile_logout));
+
+        rowsAdapter.add(new ListRow(header, adapter));
+    }
+
+    public void onItemClicked(Presenter.ViewHolder itemViewHolder, Object item, RowPresenter.ViewHolder rowViewHolder, Row row) {
+        if (!(item instanceof GridButton)) return;
+
+        switch (((GridButton) item).getId()) {
+            case LOGOUT:
+                TvApp app = TvApp.getApplication();
+
+                if (app.getIsAutoLoginConfigured()) {
```

Comment:
This may be beyond the scope of this PR, but I have been trying to understand why this behaves differently when autologin is configured. Why would we not just always show the user selection screen when logout is selected? Any ideas?

Reply:
I think this is because auto-login skips the user screen so the only way to change users is using the logout button. I'd rather have the functionality work the same no matter what your settings are. So I'll change it to always open the user screen.

In the future we might rename it to "Change User" too because it doesn't actually clear credentials.